docs/introduction.en.rst

```diff
@@ -0,0 +1,454 @@
+Introduction to online-judge-tools (English)
+=============================================
+
+online-judge-tools is a tool for automating typical tasks that exist in competitive programming.
+
+
+Install
+------------
+
+If Python is installed, you can install it with just the following command.
+
+.. code-block:: console
+
+   $ pip3 install --user online-judge-tools
+
+Linux or Mac OS is recommended for the OS, but it also works on Windows.
+
+
+Sample case test
+----------------------
+
+Do you test with sample cases before submission?
+Have you ever been troublesome and omitted?
+You should always test before submitting, since submitting an implementation that doesn't even fit the sample will only incur a penalty.
+It is also useful for debugging, and you should try the sample case every time you rewrite your program.
+
+However, "open the problem page, copy the sample input, run the program and paste it there, and compare the output result with the sample output" for the number of sample cases, it is quite troublesome to do this each time is.
+Doing the tedious work manually is easy and easy to omit or mistake.
```

Comment:
I feel uncomfortable with that translation.
We want to say that  "manual work is easy to omit or mistake, so it is not good".


How about `Doing the tedious work manually is easy to omit or mistake.`?

Reply:
Okay